test/bruno/4 - General/Create preview token.bru

```diff
@@ -0,0 +1,25 @@
+meta {
+  name: Create preview token
+  type: http
+  seq: 2.5
+}
+
+post {
+  url: {{host}}/default/rdmp/preview
+  body: none
+  auth: none
+}
+
+tests {
+  test("Status code is 200", function () {
+      expect(res.getStatus()).to.equal(200);
+  });
+  
+  test("Stores preview token", function () {
+  const raw = res.getBody();
+  const body = typeof raw === 'string' ? JSON.parse(raw) : raw;
```

Comment:
The pattern `const body = typeof raw === 'string' ? JSON.parse(raw) : raw;` is a bit off-putting - is this test expecting text or json? If json, why is `res.getBody()` not sufficient? If text or json, why can the response be either?

Reply:
The newer versions of bruno return JSON, the one we're on returns a string. I don't think this is a big deal.